terra/asset.json

```diff
@@ -1143,6 +1144,7 @@
     "decimals": "6",
     "type": "cw20",
     "circ_supply_api": "https://raw.githubusercontent.com/lioncubdao/logo/main/cub_supply",
+    "total_supply_api": "fromStaticAmount",
```

Comment:
Similarly, what's the total supply amount for CUB?

Reply:
the token supply of CUB is one trillion (1000000000000)